src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,35 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService)
+                injector.getInstance(ManufacturerService.class);
+        Manufacturer manufacturer = new Manufacturer();
+        manufacturer.setCountry("Germany");
+        manufacturer.setName("Mercedes");
+        manufacturerService.create(manufacturer);
+        manufacturer.setName("Lamborghini");
+        manufacturer.setCountry("Italy");
+        manufacturerService.update(manufacturer);
+        manufacturerService.get(2L);
+        manufacturerService.delete(1L);
```

Comment:
Hardcoding the ID to delete is not a good practice. You can use the ID of the manufacturer you created earlier.